theme/scripts/patronSlider.ts

```diff
@@ -0,0 +1,176 @@
+import * as rangeSlider from "nouislider";
+import getNextSibling from "./utils/getNextSibling";
+
+const TIER_PRICES = [5, 10, 15, 20, 25];
+
+export default (function () {
+  const PatronSlider = {
+    SELECTORS: {
+      slider: "[data-patron-slider]",
+      handle: ".noUi-handle",
+      pips: ".noUi-marker",
+      pipsValue: ".noUi-value",
+      patronTierLink: "[data-patron-tier-link]",
+      patronTierDescription: "[data-patron-tier-description]",
+      patronTierName: "[data-patron-tier-name]",
+    },
+    START_VALUE: 15,
+    MONTHLY: "Monthly",
+    PATRON_TIERS: {
```

Comment:
Up to you, but I would move these translations to a constant outside this module. Would make this module a bit smaller/easier to parse and given they are static, they don't need to be within the module itself